### DISCUSSION TITLE
dispread.exe could not be started

### DISCUSSION THREAD

**Discussion by MaxKalvan** [eoyilmaz/displaycal-py3#461] [Q&A]
I installed latest version of Displaycal on windows 11, I run it as Admin but "start measurement" in greyed out and I get the process "dispread.exe could not be started" error message, I am using Spyder4 and installed Argyll 3.3
Any help is appreciated.
Thanks
Max from Italy

**Comment by Yamh631** [comment]
Hi,

I had the same error, the problem was python 3.12, i downgraded to 3.11 and it works.
